guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-03@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-03-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-03-mobile@2x.png"
+   alt-text = "Lori routes a 400,000 satoshi payment to Lamar"
+   width = 800
+   height = 348
+%}
+
+
+It works in the other direction, too. After Lamar routes a 200,000 sats payment through the channel, Lori's outbound liquidity is 700,000 sats and her inbound liquidity is 300,000 sats.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-04.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-04@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-04-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-04-mobile@2x.png"
+   alt-text = "Lamar routes a 200,000 satoshi payment to Lori"
+   width = 800
+   height = 348
+%}
+
+Lori and Lamar can also change their liquidity by forming channels with other lightning nodes. For example, suppose Lindsay opens a channel with Lori for 2,000,000 sats. Lori’s total inbound liquidity has now increased to 2,300,000 sats, which is a combination of the inbound liquidity she has in the channel with Lindsay (2,000,000 sats) and the channel with Lamar (300,000 sats). Her outbound liquidity remains at 700,000 sats, which is a combination of the outbound liquidity she has in the channel with Lindsay (0 sats) and the channel with Lamar (700,000 sats).
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-05.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-05@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-05-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-05-mobile@2x.png"
+   alt-text = "Lindsay opens a 2,000,000 satoshi channel with Lori"
+   width = 800
+   height = 800
+%}
+
+Lori, Lamar, and Lindsay can open channels with as many other nodes as they like -- the only limitation is how much bitcoin they are willing to put into lightning channels. This interconnected web of channels is effectively “the lightning network”.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/liquidity/liquidity-06.png"
+   retina = "/assets/images/guide/how-it-works/liquidity/liquidity-06@2x.png"
+   mobile = "/assets/images/guide/how-it-works/liquidity/liquidity-06-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/liquidity/liquidity-06-mobile@2x.png"
+   alt-text = "Lori, Lamar, Lindsay, and the lightning network"
+   width = 1200
+   height = 800
+   layout = "full-width"
+%}
+
+Getting liquidity is not a one time issue on the lightning network. Routing nodes are constantly rebalancing their channels so that they have the right amount of liquidity in the right places.
```

Comment:
```suggestion
Getting liquidity is not a one time issue on the lightning network. Routing nodes are constantly rebalancing their channels so that they have the right amount of liquidity in the right places. They do this by essentially sending themselves payments through the lightning network. 
```